shell/platform/darwin/macos/framework/Source/FlutterDartProject.mm

```diff
@@ -77,4 +82,12 @@ - (NSString*)ICUDataPath {
   return arguments;
 }
 
+- (void)setDartEntrypointArguments:(NSArray*)arguments {
+  _entrypointArguments = arguments;
+}
+
+- (NSArray*)dartEntrypointArguments {
+  return _entrypointArguments;
+}
```

Comment:
You don't need to write these methods; they aren't doing anything the synthesized property methods wouldn't do.